shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -401,11 +400,7 @@ - (UITextRange*)selectedTextRange {
   return [[_selectedTextRange copy] autorelease];
 }
 
-- (void)setSelectedTextRange:(UITextRange*)selectedTextRange {
-  [self setSelectedTextRange:selectedTextRange updateEditingState:YES];
-}
-
-- (void)setSelectedTextRange:(UITextRange*)selectedTextRange updateEditingState:(BOOL)update {
+- (void)_setSelectedTextRange:(UITextRange*)selectedTextRange {
```

Comment:
In objc we don't prefix method names with `_`

Reply:
Changed to `setSelectedTextRangeLocal`